spec/requests/providers/legal_aid_applications_spec.rb

```diff
@@ -32,6 +34,17 @@
         end
       end
     end
+
+    context 'when login as other provider is authenticated' do
```

Comment:
Maybe I'm wrong but this sentence seems a bit weird to me.
Maybe "when logged in as another provider" or "when another provider is authenticated"?